src/collection-items-utils.ts

```diff
@@ -0,0 +1,20 @@
+import { DecentralandAssetIdentifier } from './types'
+
+/**
+ * Checks if the URN follows the standard ERC-721 by having the tokenId identifier at the end
+ * @public
+ */
+export function isExtendedUrn(asset: DecentralandAssetIdentifier): boolean {
+  return ['blockchain-collection-v1-item', 'blockchain-collection-v2-item'].includes(asset.type)
```

Comment:
should we perhaps move this list as a constant outside? not quite sure how smart is the vm/compiler with this